op-scim-bridge/templates/deployment.yaml

```diff
@@ -98,10 +98,10 @@ spec:
             - name: "OP_PORT"
               value: "{{ .Values.scim.httpPort }}"
             - name: "OP_SESSION"
-              {{- if .Values.scim.credentialsVolume }}
+              {{- if .Values.scim.credentialsVolume.enabled }}
               value: "/home/opuser/.op/{{ .Values.scim.credentialsVolume.file }}"
               {{- end }}
-              {{- if .Values.scim.credentialsSecret }}
+              {{- if .Values.scim.credentialsSecret.enabled }}
```

Comment:
please provide separate toggles for managing the secret and for injecting the secret ... see #87 for details